### ISSUE TITLE
file manager image thumbnails don't have text alternatives

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6812]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.1), the link in the image thumbnail on the `file manager` page does not have a text alternative and does not meet WCAG AA requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. As an admin, go to https://dev.nurax.samvera.org/concern/images/ww72bb76t/file_manager?locale=enand run the Site Improve browser extension

![file_manager_text_alternative](https://github.com/samvera/hyrax/assets/32644254/83d54225-de59-4e90-b17f-d4df16b9cb90)

**Comment by rjkati** [comment]
Note that both the link highlighted in the screenshot above **and** the image thumbnail do not have text alternatives.

**Comment by christhepianist** [comment]
Tested and confirmed that this is no longer an issue.
